src/renderer/components/ShareDialog.js

```diff
@@ -5,28 +5,9 @@ function ShareDialog(props) {
     return (
         <div className="modalCol">
             <p> Share the following link with course staff to let us access your code. </p>
-            <p> It is always safe to post this link on Piazza publicly. </p>
```

Comment:
Removed all this functionality simply because they'll be in Git if we ever decide to reenable them.